tests/test_cvat_format.py

```diff
@@ -595,3 +597,76 @@ def test_inplace_save_writes_only_updated_data(self):
                 require_media=True,
                 ignored_attrs={"frame"},
             )
+
+    @mark_requirement(Requirements.DATUM_GENERAL_REQ)
+    def test_video_export(self):
+        dataset_video = Dataset.import_from(
+            DUMMY_VIDEO_DATASET_FILE,
+            "video_frames",
+            start_frame=0,
+            end_frame=20,
+            name_pattern="frame_%06d",
+        )
+
+        items = []
+        video_hw = None
+        for frame, item in enumerate(dataset_video):
+            video_hw = item.media.size
+            if 3 < frame and frame < 15:
+                item.annotations = [
+                    Bbox(
+                        3,
+                        4,
+                        7,
+                        1,
+                        label=1,
+                        id=0,
+                        attributes={
+                            "occluded": True,
+                            "outside": False,
+                            "keyframe": True,
+                            "track_id": 0,
+                        },
+                    ),
+                    Points(
+                        [21.95, 8.00, 2.55, 15.09, 2.23, 3.16],
+                        label=0,
+                        id=1,
+                        attributes={
+                            "occluded": False,
+                            "outside": False,
+                            "keyframe": True,
+                            "track_id": 1,
+                        },
+                    ),
+                ]
+                item.attributes = {"frame": frame}
+                items.append(item)
+
+        dataset = Dataset.from_iterable(
+            items,
+            categories={
+                AnnotationType.label: LabelCategories.from_iterable(
+                    [
+                        ["label1"],
+                        ["label2"],
+                    ]
+                )
+            },
+        )
```

Comment:
I think this dataset does not include video `media` unlike `dataset_video`. Do you have any reason to make a new one rather than using `dataset_video`? I think that it is more natural to test `dataset_video` exportation.

Reply:
"Dataset.import_from (.., "video_frames", ...)" sets categories to {}.
To my best knowledge, there is no option to add categories in the initialization.
I tried 'define_categories https://github.com/openvinotoolkit/datumaro/blob/develop/datumaro/components/dataset.py#L857
but it throws CategoriesRedefinedError() https://github.com/openvinotoolkit/datumaro/blob/725110a6f1d0e37d1e2d92b194219e8a65a63c28/datumaro/components/dataset.py#L586

Reply:
Great! I changed to using ProjectLabels().
Please have a look, I couldn't sure that my changes are the best practice since I'm not familiar with 'transform'.

Reply:
As vinnam suggested, I changed to use "dataset.transform(ProjectLabels, dst_labels=["label1", "label2"])"